FreeRTOS/FreeRTOS
# Minor update to the UART write function in the IAR/QEMU/MPS2 demo

This PR updates the UART write function in the IAR/QEMU/MPS2 demo so it now checks to ensure there is space in the Tx buffer before writing to the buffer - although this does not appear to be necessary in QEMU it is more correct.

Test Steps
-----------
Not a fix - no test steps.

Related Issue
-----------
None


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
